src/NodaTime.Demo/DateIntervalDemo.cs

```diff
@@ -69,5 +69,23 @@ public void Contains_Interval()
             var result = Snippet.For(interval.Contains(june));
             Assert.AreEqual(true, result);
         }
+
+        [Test]
+        public void Deconstruction()
+        {
+            var calendar = CalendarSystem.Gregorian;
+            LocalDate start = new LocalDate(2017, 1, 1);
+            LocalDate end = new LocalDate(2017, 12, 31);
+
+            DateInterval value = new DateInterval(start, end);
+
+            value.Deconstruct(out LocalDate actualStart, out LocalDate actualEnd);
+
+            Assert.Multiple(() =>
```

Comment:
Please don't use Assert.Multiple in snippets. The snippet converter doesn't "know" how to convert it into non-test code, and it doesn't make sense to add that into the code - just multiple top-level calls to Assert.AreEqual is fine. They'll end up as multiple Console.WriteLine calls.

Reply:
Sure :) It was a bit hard to understand how the snippets will look once generated. Is there a tool for previewing the generated docs locally? Or will https://nodatime.org/unstable/api/NodaTime.DateInterval.html be updated once a release is ready? I can create a README in https://github.com/nodatime/nodatime/tree/master/src/NodaTime.Demo to clarify, if you like.

Reply:
You can clone https://github.com/nodatime/nodatime.org as a peer of this repo, then run `build/testsnippets.sh` - that's probably the best way to go for now.

The release process is a box I have here locally running the build once every half hour, but only during certain times of day. That goes against master, so the date adjusters code is now live, for example: https://nodatime.org/unstable/api/NodaTime.DateAdjusters.html

Reply:
Great, thanks! I checked `DateAdjusters` and the snippets didn't use the declared variable `end`, so I've inlined `end` instead in the asserts. The snippets should not include `end` in the next release.